packages/g/gromacs/package.yml

```diff
@@ -1,8 +1,8 @@
 name       : gromacs
-version    : '2022'
-release    : 10
+version    : '2024.2'
+release    : 11
 source     :
-    - https://ftp.gromacs.org/pub/gromacs/gromacs-2022.tar.gz : fad60d606c02e6164018692c6c9f2c159a9130c2bf32e8c5f4f1b6ba2dda2b68
+    - https://ftp.gromacs.org/pub/gromacs/gromacs-2024.2.tar.gz : 802a7e335f2e895770f57b159e4ec368ebb0ff2ce6daccf706c6e8025c36852b
 homepage   : https://www.gromacs.org/
 license    : LGPL-2.1
```

Comment:
Please update the license according to [SPDX Licenses](https://spdx.org/licenses/)

Reply:
Thanks, I completely missed that!